backend/src/main/java/com/bakdata/conquery/models/query/PrintSettings.java

```diff
@@ -52,7 +52,7 @@ public PrintSettings(boolean prettyPrint, Locale locale, DatasetRegistry dataset
 		this.prettyPrint = prettyPrint;
 		this.locale = locale;
 		this.datasetRegistry = datasetRegistry;
-		this.currency = config.getLocale().getCurrency();
+		this.currency = config.getPreprocessor().getParsers().getCurrency();
```

Comment:
macht das sinn, dass print auf preprocess config abhängt? Sollte zwar in der theorie immer die gleiche config sein aber doch etwas ungeschickt an der stelle (aber besser als 2 configs für das gleiche zu pflegen)

Reply:
Eigentlich müsste Currency in der Table/Import.json fest gelegt werden, aber das sorgt dann dafür, dass wir  unterschiedliche Currencies unterstützen